crates/core/tedge_mapper/src/mapper.rs

```diff
@@ -70,6 +70,11 @@ impl Mapper {
 
     #[instrument(skip(self), name = "messages")]
     async fn subscribe_messages(&mut self) -> Result<(), MqttClientError> {
+        let init_messages = self.converter.init_messages();
```

Comment:
Could these few lines sending the init messages be moved to a separate `init` function as this piece of logic doesn't have any specific connection to `subscribe_messages` logic?